Also, I didn't look through all of the buildbot failures, but my results above are consistent with the [raspbian failure](https://buildbot.python.org/all/#/builders/645/builds/31/steps/5/logs/stdio):

```
======================================================================
FAIL: test_uname_processor (test.test_platform.PlatformTest)
On some systems, the processor must match the output
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/buildbot/workers/3.x.gps-raspbian/build/Lib/test/test_platform.py", line 172, in test_uname_processor
    self.assertEqual(
AssertionError: '' != 'unknown'
```

--- a/Lib/test/test_platform.py
+++ b/Lib/test/test_platform.py
@@ -174,7 +174,7 @@ class PlatformTest(unittest.TestCase):
         if expect == 'unknown':
             return
 
-        self.assertEqual(expect, platform.uname().processor)
+        self.assertEqual(platform.uname().processor, expect)
 
     @unittest.skipUnless(sys.platform.startswith('win'), "windows only test")
     def test_uname_win32_ARCHITEW6432(self):

--- a/Lib/test/test_platform.py
+++ b/Lib/test/test_platform.py
@@ -172,7 +172,7 @@ class PlatformTest(unittest.TestCase):
             expect = subprocess.check_output(['uname', '-p'], text=True).strip()
 
         if expect == 'unknown':
-            return
+            expect = ''
 
         self.assertEqual(platform.uname().processor, expect)
 

